logviewer/kubelet/handlers.go

```diff
@@ -0,0 +1,116 @@
+package kubelet
```

Comment:
This is a lot of new code for the logviewer. Do we even need it?

On the stock log viewer, is the only thing we need to change the base path for where the logs are?

Everything else should be the same.

I guess this is complicated by the fact that there are two modes, proxy and non :(

Reply:
Fair point, I'll see if I can try to convince the default handlers to perform the way I want.  I may have overreacted when I saw all the string building in the base handlers.  Might have a followup PR to swap them to templates.

Reply:
As a note, this is a replacement for the logviewer we inject, not the one in titus-executor, right? I like the new, idiomatic handler. I'd prefer we add some tests to make sure it behaves 'exactly' like the log viewer we inject today, because there are platforms which might depend on its idiosyncrasies.

Reply:
Secnded, there's a lot of duplication.